apps/namadillo/src/App/Ibc/IbcFromNamada.tsx

```diff
@@ -0,0 +1,162 @@
+import { Key as KeplrKey, Window as KeplrWindow } from "@keplr-wallet/types";
+import { Panel } from "@namada/components";
+import { WindowWithNamada } from "@namada/types";
+import BigNumber from "bignumber.js";
+import { getSdkInstance } from "hooks";
+import { useAtomValue } from "jotai";
+import { useEffect, useState } from "react";
+
+import { NamCurrency } from "App/Common/NamCurrency";
+import { NamInput } from "App/Common/NamInput";
+import { accountBalanceAtom, defaultAccountAtom } from "atoms/accounts";
+import { chainAtom, chainParametersAtom } from "atoms/chain";
+import { getFirstError } from "atoms/utils";
+
+const keplr = (window as KeplrWindow).keplr!;
+const namada = (window as WindowWithNamada).namada!;
```

Comment:
For testing I had to move this inside `submitIbcTransfer` otherwise it was sometimes undefined :P